### PR TITLE
fix(datetime-picker): update accessibility

### PR DESCRIPTION
**Pull request by kosol-komenek-rft**
Description
Datetime picker change base class from ControlElement to FormFieldElement.

Fixes # (issue) https://jira.refinitiv.com/browse/ELF-2348

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by Theeraphat-Sorasetsakul — DISMISSED** [review]
LGTM.

**Comment by bualoy-napat** [comment]
Verified!

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]
It seems this PR faces the same issue as described in https://github.com/Refinitiv/refinitiv-ui/pull/1079#pullrequestreview-1848129293. Let's address it before merge.

File: packages/elements/src/datetime-picker/locales.ts [hunk 1]
```diff
@@ -14,27 +14,24 @@ import zhCN from 'date-fns/esm/locale/zh-CN/index.js';
...
+import translations from '@refinitiv-ui/phrasebook/locale/en/datetime-picker.js';
 
 // This file is a transition between using date-fns and Intl object to format dates
 // As of now, use Phraseboook to just resolve languages and locales
 // and match against the date-fns locales.
 
 // match locales against date-fns
 // This will be used with resolveLocale function
-const globals = {};
+// for 5 langauge is set in Phrasebook de / en / ja / zh / zh-hant
```
Comment (wattachai-lseg) [inline packages/elements/src/datetime-picker/locales.ts]:
```suggestion
// de, en, ja, zh & zh-hant have been defined in Phrasebook already.
```

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**